src/features/user/GiftFunds/GiftFunds.module.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../theme/theme';
+
+.allGiftFundscontainer {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+.container {
+  
+    background-color: $backgroundColor;
+    padding: 34px;
+    max-width: 700px;
+    border-radius: 9px;
+    box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%),
+    0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%);
+
+    .container_heading {
+        padding-bottom: 16px;
+    }
+    .container_details {
+        display: flex;
+        flex-direction: row;
+        margin-top: 24px;
+        flex-wrap: wrap;
+        gap: 15px;
+        div {
+            flex:1;
+        }
+    }
+    hr {
+        border: 1px solid #DDDBDA;
+    }
+}
```

Comment:
It may be good to refactor this slightly to use styled css in a similar manner to the other features added to the dashboard recently. (At a later date)